### PR TITLE
Recording the activity of "none" or "i don't" in addition to given se…

### PR DESCRIPTION
**Pull request by wozzer72**
…t of jobs, for each of Vacancies, Starters and Leavers.

Trello card: https://trello.com/c/d4uwAajW

This covers the backend changes to received and recall the options.
This covers the frontend changes to initialise UI with recorded values and to post back always on None and Don't Know options with the revised API values.

### CODE REVIEW

**Comment by wozzer72** [comment]
[Vacancies/Starters/Leavers: Need to record the 'I have no ...... answer](https://trello.com/c/d4uwAajW/188-vacancies-starters-leavers-need-to-record-the-i-have-no-answer)

File: server/models/api/jobs.js [hunk 1]
```diff
@@ -32,33 +33,41 @@ exports.jobsByTypeJSON = (givenJobs) => {
...
-  if (typeof jobTypeGroup['Vacancies'] === 'undefined') {
+  // handle each of Vacancies, Starters and Leavers
+  if (establishment.vacancies && establishment.vacancies != 'With Jobs') {
+    jobTypeGroup['Vacancies'] = establishment.vacancies;
+  } else if (typeof jobTypeGroup['Vacancies'] === 'undefined') {
     jobTypeGroup['Vacancies'] = [];
   }
-  if (typeof jobTypeGroup['Starters'] === 'undefined') {
+
+  console.log("WA DEBUG: establishment starters: ", establishment.starters)
```
[the remark this answers is not in the document]
Comment (wozzer72) [reply server/models/api/jobs.js]: Good catch, but yes, I will (and have) removed.

[the remark this answers is not in the document]
Comment (denny13) [reply server/models/api/jobs.js]: Great :)

**Comment by wozzer72** [comment]
Approved from Shakir (DB - another PR), approved by Nasir for backend and approved by Denny for frontend. This extends from an earlier online demo including Ann (tester).

Ready to merge to DEV, but only once having applied database changes to `sfcdevdb`. Now done too.

**Pull request merged**